samples/camera/README.rst

```diff
@@ -0,0 +1,34 @@
+.. _camera:
+
+CAMERA: camera
+#####################################################
+
+Description
+***********
+
+References
+**********
+
+Wiring
+*******
+
+Building and Running
+********************
+
+This project demos camera samples.
+It should work with any platform featuring a I2C peripheral interface and a CMOS sensor interface.
```

Comment:
The code seems to be very hardware-specific.  Will it really work with any CMOS sensor and board with an I2C interface?

Reply:
The sample code and framework are designed for generic camera features. But need more camera drivers and cmos sensors added to test.

Reply:
OK, thanks for explaining.